src/firefly/js/ui/SearchPanel.jsx

```diff
@@ -138,10 +147,28 @@ function SearchGroup({group, activeSearch}) {
     );
 }
 
+function RouterChange({onClick, ttips, clsname, label}) {
+    const searchPanelState = useStoreConnector(getSearchInfo);
+    const navigate = useNavigate();
+
+    const {activeSearch, allSearchItems = {}} = searchPanelState;
+    const searchItem = allSearchItems[activeSearch];
```

Comment:
`RouterChange` is not a good name.  It should be something that indicate it's a presentation of a Router's SearchItem.
Being that it's a presenter, why does it need to watch for data changes?
`    const searchPanelState = useStoreConnector(getSearchInfo);
`
All of the needed data should already be available from `SearchItem`.